.travis.yml

```diff
@@ -8,7 +8,7 @@ dist: trusty
 jdk:
 - openjdk8
 install: true # skips travis' default installation step which executes gradle assemble.
-before_script: ./set-solutioninfo-version.sh
+before_script: ./gradlew injectVersion
```

Comment:
why not make it as a part of build tasks graph (e.g. using `dependsOn`, `must/shouldRunAfter`, `finalizedBy` and so on).

Reply:
Because I only want the release version to be replaced only on travis, but not locally on every dev's machine when they run the build locally.

Reply:
then i'd recommend to create a custom gradle task (for example, `ciBuild`) and run it from travis ([here instead of `clean build`](https://github.com/commercetools/commercetools-sync-java/blob/e6b6791d27e0e42febebf57b98d093adc0039267/.travis.yml#L12), avoiding `before_script` block

Reply:
It is already a custom gradle task. I made it inline now: 62bddb4, but also applies to the `after_success` then, no?

Reply:
>but also applies to the after_success

good point!